diff -r eca32331b116 stable/resources/ffmpeg/ffmpeg-0.5/libavcodec/x86/dsputil_mmx.h
--- a/resources/ffmpeg/ffmpeg-0.5/libavcodec/x86/dsputil_mmx.h	Thu May 21 19:20:11 2009 +0300
+++ b/resources/ffmpeg/ffmpeg-0.5/libavcodec/x86/dsputil_mmx.h	Sat May 23 11:57:16 2009 +0300
@@ -32,28 +32,28 @@
 
 extern const uint64_t ff_pdw_80000000[2];
 
-extern const uint64_t ff_pw_3;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pw_3;
 extern const uint64_t ff_pw_4;
 extern const xmm_reg  ff_pw_5;
-extern const xmm_reg  ff_pw_8;
+extern const xmm_reg  __attribute__((visibility("hidden"))) ff_pw_8;
 extern const uint64_t ff_pw_15;
 extern const xmm_reg  ff_pw_16;
-extern const uint64_t ff_pw_20;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pw_20;
 extern const xmm_reg  ff_pw_28;
 extern const xmm_reg  ff_pw_32;
 extern const uint64_t ff_pw_42;
-extern const xmm_reg  ff_pw_64;
+extern const xmm_reg  __attribute__((visibility("hidden"))) ff_pw_64;
 extern const uint64_t ff_pw_96;
-extern const uint64_t ff_pw_128;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pw_128;
 extern const uint64_t ff_pw_255;
 
-extern const uint64_t ff_pb_1;
-extern const uint64_t ff_pb_3;
-extern const uint64_t ff_pb_7;
-extern const uint64_t ff_pb_1F;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pb_1;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pb_3;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pb_7;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pb_1F;
 extern const uint64_t ff_pb_3F;
-extern const uint64_t ff_pb_81;
-extern const uint64_t ff_pb_A1;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pb_81;
+extern const uint64_t __attribute__((visibility("hidden"))) ff_pb_A1;
 extern const uint64_t ff_pb_FC;
 
 extern const double ff_pd_1[2];
diff -r eca32331b116 stable/resources/ffmpeg/ffmpeg-0.5/libavcodec/x86/vc1dsp_mmx.c
--- a/resources/ffmpeg/ffmpeg-0.5/libavcodec/x86/vc1dsp_mmx.c	Thu May 21 19:20:11 2009 +0300
+++ b/resources/ffmpeg/ffmpeg-0.5/libavcodec/x86/vc1dsp_mmx.c	Sat May 23 11:57:16 2009 +0300
@@ -67,7 +67,7 @@
     "movq      %%mm"#R1", "#OFF"(%1)   \n\t"    \
     "add       %2, %0                  \n\t"
 
-DECLARE_ALIGNED_16(const uint64_t, ff_pw_9) = 0x0009000900090009ULL;
+DECLARE_ALIGNED_16(const uint64_t __attribute__((visibility("hidden"))), ff_pw_9) = 0x0009000900090009ULL;
 
 /** Sacrifying mm6 allows to pipeline loads from src */
 static void vc1_put_ver_16b_shift2_mmx(int16_t *dst,
