Index: configure
===================================================================
--- configure	(revision 16033)
+++ configure	(working copy)
@@ -1127,7 +1127,7 @@
 vhook="default"
 
 # build settings
-add_cflags -D_ISOC99_SOURCE -D_POSIX_C_SOURCE=200112
+add_cflags -D_ISOC99_SOURCE
 SHFLAGS='-shared -Wl,-soname,$$(@F)'
 VHOOKSHFLAGS='$(SHFLAGS)'
 FFSERVERLDFLAGS=-Wl,-E
@@ -1459,8 +1459,6 @@
         SHFLAGS='-shared -Wl,--output-def,$$(@:$(SLIBSUF)=.def) -Wl,--enable-runtime-pseudo-reloc -Wl,--enable-auto-image-base'
         objformat="win32"
         enable dos_paths
-        check_cpp_condition _mingw.h "(__MINGW32_MAJOR_VERSION > 3) || (__MINGW32_MAJOR_VERSION == 3 && __MINGW32_MINOR_VERSION >= 15)" ||
-            die "ERROR: MinGW runtime version must be >= 3.15."
         enabled_any avisynth vfwcap_demuxer &&
             { check_cpp_condition w32api.h "(__W32API_MAJOR_VERSION > 3) || (__W32API_MAJOR_VERSION == 3 && __W32API_MINOR_VERSION >= 13)" ||
               die "ERROR: avisynth and vfwcap_demuxer require w32api version 3.13 or later."; }
Index: libavcodec/utils.c
===================================================================
--- libavcodec/utils.c	(revision 16033)
+++ libavcodec/utils.c	(working copy)
@@ -25,8 +25,10 @@
  * utils.
  */
 
-/* needed for mkstemp() */
+/* needed for mkstemp() but breaks compatibility in Leopard */
+#ifndef __APPLE__
 #define _XOPEN_SOURCE 600
+#endif
 
 #include "libavutil/avstring.h"
 #include "libavutil/integer.h"
